ghost/members-api/lib/controllers/RouterController.js

```
@@ -645,3 +646,22 @@
         }
     }
 };
+
+function parsePersonalNote(rawText) {
+    let sanitizeHtml = require('sanitize-html');
+    if (rawText && typeof rawText !== 'string' ) {
```

ESLint

[error] There should be no space before this paren.

(space-in-parens)